backend/dao-service/src/proposal-service/proposal-service.service.ts

```diff
@@ -39,17 +39,13 @@ export class ProposalServiceService {
   }
 
 
-  async findOne(id: number, sub: string): Promise<any> {
+  async findAllProposals(sub: string): Promise<any> {
     const role = await this.getUserRole(sub);
     console.log(role);
     if (role != 'MFS') {
       throw new UnauthorizedException("User does not have permission");
```

Comment:
Before Throwing error make sure to keep a log here so the distributed logging project can have a environment to test out.

Reply:
This has been fixed